optimize/backend/src/it/java/io/camunda/optimize/test/optimize/VariablesClient.java

```diff
@@ -48,19 +42,20 @@ public VariablesClient(final Supplier<OptimizeRequestExecutor> requestExecutorSu
             .enable(SerializationFeature.INDENT_OUTPUT);
   }
 
-  public List<ProcessVariableNameResponseDto> getProcessVariableNames(
-      ProcessDefinitionEngineDto processDefinition) {
-    ProcessToQueryDto processToQuery = new ProcessToQueryDto();
-    processToQuery.setProcessDefinitionKey(processDefinition.getKey());
-    processToQuery.setProcessDefinitionVersions(
-        ImmutableList.of(processDefinition.getVersionAsString()));
-
-    List<ProcessToQueryDto> processesToQuery = List.of(processToQuery);
-    ProcessVariableNameRequestDto variableRequestDto =
-        new ProcessVariableNameRequestDto(processesToQuery);
-
-    return getProcessVariableNames(variableRequestDto);
-  }
+  //
+  //  public List<ProcessVariableNameResponseDto> getProcessVariableNames(
+  //      ProcessDefinitionEngineDto processDefinition) {
+  //    ProcessToQueryDto processToQuery = new ProcessToQueryDto();
+  //    processToQuery.setProcessDefinitionKey(processDefinition.getKey());
+  //    processToQuery.setProcessDefinitionVersions(
+  //        ImmutableList.of(processDefinition.getVersionAsString()));
+  //
+  //    List<ProcessToQueryDto> processesToQuery = List.of(processToQuery);
+  //    ProcessVariableNameRequestDto variableRequestDto =
+  //        new ProcessVariableNameRequestDto(processesToQuery);
+  //
+  //    return getProcessVariableNames(variableRequestDto);
+  //  }
```

Comment:
not sure I understand why we comment this out here?

Reply:
Hmm looks like I commented this out to see if `DecisionDefinitionEngineDto` can easily be removed else and then forgot to follow up with the respective removal 😬

Reply:
Badly commented from my side - wanted to remove DecisionDefinitionEngineDto but keep the helper methods for referral when migrating IT because we weren't sure yet if we're keeping decision logic. Now that we're removing it this should have also been removed